### PR TITLE
Adds pending contribution table and modal

### PR DESCRIPTION
**Pull request by NejcZdovc**
## Changes
- add new dialog
image

- add new element to wallet summary
image

##### Link / storybook path to visual changes
- https://brave-ui-gn3z9nt5n.now.sh

## Integration
- [ ] Does this contain changes to src/components or src/
  - [ ] Will you publish to npm immediately after this PR, or wait until sometime in the future?
  - [ ] Incompatible API change to something existing _(major version increase)_
  - [ ] Adding new backwards-compatible functionality? _(minor version increase)_
  - [ ] Fixing a bug backwards-compatibly? _(patch version increase)_
  
- [ ] Does this contain changes to src/features for brave-core?
  - [ ] Are there non backwards-compatible changes required for brave-core? **Do not merge until brave-core PR is approvable.** Link to brave-core PR:
  - [ ] Will you create brave-core PR to update to this commit after it is merged?
  - [ ] Wants uplift to brave-core feature branch?
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit

### CODE REVIEW

File: src/features/rewards/tablePending/style.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  border: none;
+  background: none;
+  cursor: pointer;
+  text-align: center;
+  width: 100%;
+`
+
+export const StyledRemoveIcon = styled<{}, 'span'>('span')`
+  vertical-align: middle;
+  color: #9E9FAB;
```
Comment (ryanml) [inline src/features/rewards/tablePending/style.ts]: Do we have a value from the palette we can grab for this one? (Ditto for the one in `StyledRemove`)

Comment (NejcZdovc) [reply src/features/rewards/tablePending/style.ts]: this color is not in the panel, we could find similar one

Comment (ryanml) [reply src/features/rewards/tablePending/style.ts]: `#A0A1B2` (grey400) seems to look pretty close

Current:
screen shot 2019-02-05 at 12 17 37 am

grey400:
screen shot 2019-02-05 at 12 17 50 am

Comment (NejcZdovc) [reply src/features/rewards/tablePending/style.ts]: done

**Pull request merged**